### ISSUE TITLE
AttributeError: 'LangchainLLMWrapper' object has no attribute 'embed_query'

### ISSUE THREAD

**Issue by cosmin-novac** [explodinggradients/ragas#529]
**Describe the bug**
Evaluate doesn't work due to compatibility issues with LangchainLLMWrapper, regardless if I wrap the models or not.

Ragas version: 0.0.23.dev52+g08dd0a7
Python version: 3.11.7

**Error trace**
 File "C:\Users\<USER>\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\ragas\metrics\_answer_relevance.py", line 85, in calculate_similarity
    question_vec = np.asarray(self.embeddings.embed_query(question)).reshape(1, -1)
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'LangchainLLMWrapper' object has no attribute 'embed_query'

**Infos**
These are the models used in the eval:

```
metrics=[
    faithfulness, # requires col "contexts"
    answer_relevancy, # requires col "contexts"
    harmfulness, #['contexts']
]

result = evaluate(
    embeddings=embeddings_az,
    llm=llm_az,
    dataset=dataset.select(range(1)),
    metrics= metrics
)
```

```
embeddings_az
AzureOpenAIEmbeddings(client=<openai.resources.embeddings.Embeddings object at 0x000001D8D63E5F10>, async_client=<openai.resources.embeddings.AsyncEmbeddings object at 0x000001D8D66088D0>, model='text-embedding-ada-002', deployment='text-embedding-ada-002', openai_api_version='2023-05-15', openai_api_base=None, openai_api_type='azure', openai_proxy='', embedding_ctx_length=8191, openai_api_key='...', openai_organization=None, allowed_special=set(), disallowed_special='all', chunk_size=16, max_retries=2, request_timeout=None, headers=None, tiktoken_enabled=True, tiktoken_model_name=None, show_progress_bar=False, model_kwargs={}, skip_empty=False, default_headers=None, default_query=None, retry_min_seconds=4, retry_max_seconds=20, http_client=None, azure_endpoint='https://....openai.azure.com/', azure_ad_token=None, azure_ad_token_provider=None, validate_base_url=True)

llm_az
AzureOpenAI(client=<openai.resources.completions.Completions object at 0x000001D8D6383810>, async_client=<openai.resources.completions.AsyncCompletions object at 0x000001D8D63BCA50>, openai_api_key='...', openai_proxy='', azure_endpoint='https://....openai.azure.com/', deployment_name='gpt-35-turbo-tw', openai_api_version='2023-05-15', openai_api_type='azure')
```

**Comment by jjmachan** [comment]
hey @tenshis 
apologies for the late reply but have you tried wrapping the embeddings like this before passing

```py
from ragas.embeddings.base import LangchainEmbeddingsWrapper

embeddings = LangchainEmbeddingsWrapper(embeddings_az)
``` 
and then pass it

can you check this and let me know if it works?

**Comment by tenshis** [comment]
Hello jjmachan,

Thanks for the answer. I'm afraid there is no LangchainEmbeddingsWrapper in ragas.embeddings.base? 
I am on version 0.1.0rc1 .

**Comment by jjmachan** [comment]
oh you have to install from source for that, I've merged the fix #553
